src/components/UNSTABLE__Shell/ShellReturnToBanner/index.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * @file 'Return' navigation.
+ * @copyright IBM Security 2020
+ */
+
+import ArrowLeft16 from '@carbon/icons-react/lib/arrow--left/16';
+
+import dataUri from 'data-uri.macro';
+import { node, string } from 'prop-types';
+import React from 'react';
+
+import { appendComponentNamespace } from '../../../globals/namespace';
+
+import Icon from '../../Icon';
+import Link from '../../Link';
+
+import {
+  legacyNamespace as legacyShellNamespace,
+  namespace as shellNamespace,
+} from '../Shell';
+
+const suffix = 'banner';
+const legacyNamespace = appendComponentNamespace(legacyShellNamespace, suffix);
+
+function ShellReturnToBanner({ children, href, labelText, view, ...other }) {
+  return (
+    <Link
+      className={`${legacyNamespace} ${appendComponentNamespace(
+        shellNamespace,
+        suffix
+      )}`}
+      href={href}
+      style={{
+        backgroundImage: `url(${dataUri(
+          '../../../images/aurora-banner@2x.png'
+        )})`,
+      }}
+      {...other}
+    >
+      <Icon className={`${legacyNamespace}__icon`} renderIcon={ArrowLeft16} />
```

Comment:
Just an idea -- instead of `Link` with an `Icon` inside (likely involves adding placement styles), you can use --

```jsx
<Button
  href={href}
  renderIcon={ArrowLeft16}
  {...etc}
/>
```

The `href` prop turns it into a semantic `a`/anchor element & then the icon placement is handled for you.

Reply:
Good thought! I originally tried this and it didn't play so nice out-of-the-box - 

<img width="1470" alt="Button" src="https://user-images.githubusercontent.com/3846874/75439470-c163fd00-5951-11ea-8e8d-b2a6e7255a5d.png">

To remain close to the original implementation and to position the icon on the left as per designs, the `Link` seemed more appropriate:

<img width="1472" alt="Link" src="https://user-images.githubusercontent.com/3846874/75439465-bf01a300-5951-11ea-9ba1-ef57e919d0d9.png">